gamification/src/main/java/com/appcoins/wallet/gamification/repository/entity/GamificationStatus.kt

```diff
@@ -1,6 +1,16 @@
 package com.appcoins.wallet.gamification.repository.entity
 
 enum class GamificationStatus {
-  NONE, STANDARD, APPROACHING_NEXT_LEVEL, APPROACHING_VIP, VIP, APPROACHING_VIP_MAX, VIP_MAX
+  NONE,
+  STANDARD,
+  APPROACHING_NEXT_LEVEL,
+  APPROACHING_VIP,
+  VIP,
+  APPROACHING_VIP_MAX,
+  VIP_MAX;
+
+  companion object {
+    fun toEnum(value: String?) = values().firstOrNull {it.name == value} ?: NONE
```

Comment:
hmm to guarantee it doesn't throw an exception when the element doesn't exist, it would need to be:
```
try {
      GamificationStatus.valueOf(value ?: "")
    } catch (e: IllegalArgumentException) {
      NONE
    }
```
Do you prefer like this?

Reply:
Let it be as it is no, no problem